std_msgs/Header header
sensor_msgs/PointCloud2 object_configuration
geometry_msgs/Pose[] rope_node_transforms
string[] gripper_names
geometry_msgs/Pose[] gripper_poses
float64[] robot_configuration
bool robot_configuration_valid
geometry_msgs/Point[] gripper_nearest_point_to_obstacle
geometry_msgs/Vector3[] obstacle_surface_normal
float64[] gripper_distance_to_obstacle
float64 sim_time
